ethereum/core/src/main/java/org/hyperledger/besu/ethereum/privacy/PrivacyController.java

```diff
@@ -100,5 +100,8 @@ void verifyPrivacyGroupContainsEnclavePublicKey(
 
   PrivateTransactionSimulator getTransactionSimulator();
 
-  Optional<Hash> getBlockHashByBlockNumber(Optional<Long> blockNumber);
+  Optional<Hash> getBlockHashByBlockNumber(final Long blockNumber);
```

Comment:
Do we need to expose this method in this interface? Isn't it used internally?

I recon we could just find the blockhash from the number inside `getStateRootByBlockNumber()` method.